server/service.go

```diff
@@ -484,7 +537,8 @@ func (m *BoostService) processBellatrixPayload(w http.ResponseWriter, req *http.
 	relays := originalBid.relays
 	if len(relays) == 0 {
 		log.Warn("originating relay not found, sending getPayload request to all relays")
-		relays = m.relays
+
+		relays = m.relayConfigurator.AllRelays()
```

Comment:
Can we remove some of the newly-added blank lines in this file? To me it looks wrong.

Reply:
Not really, otherwise the linters will complain.  Statements and assignments must not be coupled.

Reply:
Which linter complains about that?

I don't see any complains about that in `make lint` with these changes:

<details>
<summary>Diff</summary>

```diff
diff --git a/server/service.go b/server/service.go
index 1ffbfb0..0841798 100644
--- a/server/service.go
+++ b/server/service.go
@@ -295,7 +295,6 @@ func (m *BoostService) handleRegisterValidator(w http.ResponseWriter, req *http.
 		if len(relays) < 1 {
 			log.Warnf("there are no relays specified for %s", pubKey)
 		}
-
 		relayPayloads.add(relays, p)
 	}
 
@@ -307,15 +306,11 @@ func (m *BoostService) handleRegisterValidator(w http.ResponseWriter, req *http.
 
 		go func(r relay.Entry, payloads []types.SignedValidatorRegistration) {
 			defer wg.Done()
-
 			relayURL := r.GetURI(pathRegisterValidator)
-
 			_, err := SendHTTPRequest(context.Background(), m.httpClientRegVal, http.MethodPost, relayURL, ua, payloads, nil)
 			relayErrCh <- err
-
 			if err != nil {
 				log.WithField("url", relayURL).WithError(err).Warn("error calling registerValidator on relay")
-
 				return
 			}
 		}(r, payloads)
@@ -331,7 +326,6 @@ func (m *BoostService) handleRegisterValidator(w http.ResponseWriter, req *http.
 	for err := range relayErrCh {
 		if err == nil {
 			m.respondOK(w, nilResponse)
-
 			return
 		}
 	}
@@ -543,7 +537,6 @@ func (m *BoostService) processBellatrixPayload(w http.ResponseWriter, req *http.
 	relays := originalBid.relays
 	if len(relays) == 0 {
 		log.Warn("originating relay not found, sending getPayload request to all relays")
-
 		relays = m.relayConfigurator.AllRelays()
 	}
 
@@ -558,7 +551,6 @@ func (m *BoostService) processBellatrixPayload(w http.ResponseWriter, req *http.
 
 	for _, r := range relays {
 		wg.Add(1)
-
 		go func(relay relay.Entry) {
 			defer wg.Done()
 			url := relay.GetURI(pathGetPayload)
@@ -778,7 +770,6 @@ func (m *BoostService) CheckRelays() int {
 
 	for _, r := range m.relayConfigurator.AllRelays() {
 		wg.Add(1)
-
 		go func(relay relay.Entry) {
 			defer wg.Done()
 			url := relay.GetURI(pathStatus)
```
</details>

Reply:
Sorry for confusion. I was running a much stricter set of linters locally on the newly introduced changes via pre-commit hook. `wsl` was included.

Reply:
Also could me tell me if the project has a style guide where it's recommended to no use blank lines?

Reply:
There is no well-defined style guide, it just needs to pass the formatting checks:

https://github.com/flashbots/mev-boost/blob/3b7f08b6e2c38730b7fc37edc73b147a1d5df0c8/Makefile#L56-L58

Generally, this project leans towards denser code with less whitespace.

Reply:
I'm not necessarily against that style, it's just a little inconsistent with the other functions in that file.

Reply:
The code passed the checks, didn't it? I'll remove the line to save us some time for some more important things than a holy war on blank lines :)

Reply:
what's the case for adding a bunch of new newlines? that's generally not considered best practice, i'd say :)

Reply:
That’s a strong assertion, as some people and linters will disagree. However if you want me to remove the blank lines, I will do that. As I said, with no style guide in place this is just a matter of taste. Please let me know if there are any other points that should be addressed as I’d rather fix them all at once.

Reply:
Fixed, empty lines removed.